### PR TITLE
bisector: try older commits if old_commit has the same result as new_commit

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR

File: infra/bisector.py [hunk 1]
```diff
@@ -212,8 +214,8 @@ def _bisect(bisect_type, old_commit, new_commit, test_case_path, fuzz_target,
       logging.warning('new_commit crashed but not shouldn\'t. '
                       'Continuing to see if stack changes.')
 
-    # Check if the error is persistent through the commit range
-    if old_commit:
+    range_valid = False
+    for _ in range(2):
```
Comment (jonathanmetzman) [inline infra/bisector.py:218]: Why do this? I think it should have a comment.

File: infra/bisector.py [hunk 2]
```diff
@@ -225,7 +227,23 @@ def _bisect(bisect_type, old_commit, new_commit, test_case_path, fuzz_target,
 
       if _check_for_crash(build_data.project_name, fuzz_target,
                           test_case_path) == expected_error:
-        raise BisectError('old_commit had same result as new_commit', repo_url)
+        logging.warning('old_commit %s had same result as new_commit %s',
+                        old_commit, new_commit)
+        # Try again on an slightly older commit.
```
Comment (jonathanmetzman) [inline infra/bisector.py:232]: nit: "on a slightly"

File: infra/repo_manager.py [hunk 3]
```diff
@@ -109,7 +109,21 @@ def get_current_commit(self):
       The current active commit SHA.
     """
     out, _, _ = self.git(['rev-parse', 'HEAD'], check_result=True)
-    return out.strip('\n')
+    return out.strip()
+
+  def get_parent(self, commit, count):
```
Comment (jonathanmetzman) [inline infra/repo_manager.py:114]: maybe just name this variable "n" instead of count and the comment will be easier to understand.

**Comment by jonathanmetzman** [comment]
LGTM

**Pull request merged**